_posts/2022-01-10-reassign-partitions.md

```diff
@@ -314,50 +326,50 @@ bin/kafka-reassign-partitions.sh --bootstrap-server <CLUSTER-NAME>-kafka-bootstr
  --execute
 ```
 
-You can use the `--verify` mode to check if the partition reassignment is done or if it is still running.
+You can use the `--verify` action to check if the partition reassignment is done or if it is still running. You might have to run this command multiple times since it may take the process a while to get complete.
 
 ```sh
 bin/kafka-reassign-partitions.sh --bootstrap-server <CLUSTER-NAME>-kafka-bootstrap:9092 \
   --reassignment-json-file /tmp/reassignment.json \
   --verify
 ```
-The `--verify` mode reports if the assignment is done or not.
+The `--verify` action reports if the assignment is done or not.
 
 ```shell
 Status of partition reassignment:
-Reassignment of partition my-topic-1-0 is complete.
-Reassignment of partition my-topic-1-1 is complete.
-Reassignment of partition my-topic-1-2 is complete.
-Reassignment of partition my-topic-1-3 is complete.
-Reassignment of partition my-topic-1-4 is complete.
-Reassignment of partition my-topic-1-5 is complete.
-Reassignment of partition my-topic-1-6 is complete.
-Reassignment of partition my-topic-1-7 is complete.
-Reassignment of partition my-topic-1-8 is complete.
-Reassignment of partition my-topic-1-9 is complete.
+Reassignment of partition my-topic-two-0 is complete.
+Reassignment of partition my-topic-two-1 is complete.
+Reassignment of partition my-topic-two-2 is complete.
+Reassignment of partition my-topic-two-3 is complete.
+Reassignment of partition my-topic-two-4 is complete.
+Reassignment of partition my-topic-two-5 is complete.
+Reassignment of partition my-topic-two-6 is complete.
+Reassignment of partition my-topic-two-7 is complete.
+Reassignment of partition my-topic-two-8 is complete.
+Reassignment of partition my-topic-two-9 is complete.
 
 Clearing broker-level throttles on brokers 0,1,2,3,4
-Clearing topic-level throttles on topic my-topic-1  
+Clearing topic-level throttles on topic my-topic-two  
 ```
 
-When the partition reassignment is complete, the scaled down brokers will have no assigned partitions and can be removed safely. You can check it by looking at the partition details of the reassigned topic.
+When the partition reassignment is complete, the brokers we want to scale down will have no assigned partitions and can be removed safely. You can check it by looking at the partition details of the reassigned topic.
```

Comment:
If new topics are created before these brokers are removed from the cluster, partitions may be assigned to these brokers. Should we mention it?